languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/pt-BR/grammar.xml

```diff
@@ -887,34 +889,103 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
 
     <!-- CONFUNDIR DUZENTOS GRAMAS NUMERAL COM DUZENTAS GRAMAS -->
 
-    <rule id="DUZENTAS_GRAMAS_NUMERAL" name="Duzentos gramas">
-      <pattern>
-        <token>duzentas</token>
-        <token>gramas</token>
-      </pattern>
-      <message>Se grama se refere a peso substitua &quot;duzentas gramas&quot; por <suggestion>duzentos gramas</suggestion>.</message>
-      <example correction="duzentos gramas">Comprei <marker>duzentas gramas</marker> de carne.</example>
+    <rule id="DUZENTAS_GRAMAS_NUMERAL" name="Duzentos gramas" default="temp_off">
+        <pattern>
+            <token regexp="yes">(duz|trez|quatroc|quinh|seic|setec|oitoc|novec)entas</token>
+            <token>gramas</token>
```

Comment:
Would it be too risky to broaden the numbers even more here? `Grass` is not really countable, is it? I feel like if you want to count it, `gramado` is more suitable (possibly a style rule idea?). Anyway, just some thoughts.

Reply:
Umm... what 'grass' are you referring to? Isn't this about 'grams' as the unit of mass? I personally feel like 'trezentas gramas' isn't particularly less likely to occur than the original 'duzentas gramas', so I just expanded to the other numbers between 300 and 900.

Reply:
Yes, this is about units, but the point of the rule is to avoid confusion between the two types of `gramas`, isn't it? 

I assumed the matched numbers had been limited to avoid flagging correct sentences where the user actually meant to say `grass` instead of `grams` (e.g. `meu bebê comeu três gramas e um punhado de terra`). 

But apart from cases where there is a determiner (e.g. `Vou colocar uma grama no meu quintal`), that just seems super unlikely to occur. So I wonder if we could attempt expanding the matches and see where that leads.

Reply:
I'm thinking of recipes and calorie counting texts, but maybe I'm overthinking this. Anyway, just a thought. The rule is fine.

Reply:
As far as I understood it, the point of the rule was to fix the agreement issue, as 'grama' (the unit) is masculine, not feminine. It didn't even cross my mind that anyone would be counting blades of grass. We can just roll it out and see if it affects anyone – like you, I seriously doubt it. There aren't __that many__ more numerals that would require agreement here. Maybe anything ending with '1' and '2' (for 'vinte e uma gramas' or 'vinte e duas gramas') – but that's not the kind of numeral that's very likely to be written out in full, so I wanted to leave it restricted only to do the full forms of numerals between 200 and 900, which seem more likely.